components/molecules/HighlightCard/highlight-card.tsx

```diff
@@ -154,7 +160,17 @@ const HighlightCard: React.FC<HighlightCardProps> = ({
               {getIcon(icon)}
             </div>
             {/* Label: Text */}
-            <div className="text-sm text-slate-600   leading-none">{label ? label : "Label"}</div>
+            <div className={clsx("text-sm text-slate-600 leading-none")}>
+              {tooltip ? (
+                <Tooltip className="w-44 py-3 text-center" content={tooltip}>
+                  {label}
+                </Tooltip>
+              ) : label ? (
+                label
+              ) : (
+                "Label"
+              )}
```

Comment:
refactor (blocking): The `tooltip` and `label` props are both optional. It's unclear why if there is no `tooltip` or `label` prop that the word `Label` is returned.

```suggestion
              {tooltip && label ? (
                <Tooltip className="w-44 py-3 text-center" content={tooltip}>
                  {label}
                </Tooltip>
              ) : label ? (
                label
              ) : (
                "Label"
              )}
```